TestCases/compliance-level-3/0004-lending/0004-lending.dmn

```diff
@@ -843,17 +843,17 @@
             <formalParameter typeRef="number" name="CreditScore"/>
             <decisionTable hitPolicy="UNIQUE" outputLabel="Post-bureauRiskCategoryTable" preferredOrientation="Rule-as-Row">
                 <input id="_0859d2fc-bb13-4eaa-bebf-4bf8f0866c49" label="ExistingCustomer">
-                    <inputExpression typeRef="string">
+                    <inputExpression typeRef="boolean">
                         <text>ExistingCustomer</text>
                     </inputExpression>
                 </input>
                 <input id="_fba27bc9-7678-4457-893e-6bc96fc69626" label="ApplicationRiskScore">
-                    <inputExpression typeRef="string">
+                    <inputExpression typeRef="number">
                         <text>ApplicationRiskScore</text>
                     </inputExpression>
                 </input>
                 <input id="_b54042e3-1fe7-4fb3-90d6-ec528047fafb" label="CreditScore">
-                    <inputExpression typeRef="string">
+                    <inputExpression typeRef="number">
```

Comment:
LGTM the BKM params are defined as:
```
            <formalParameter typeRef="boolean" name="ExistingCustomer"/>
            <formalParameter typeRef="number" name="ApplicationRiskScore"/>	            
            <formalParameter typeRef="number" name="CreditScore"/>	            
```
and as the decision table is using them directly, so `boolean`, `number`, `number`